source/postprocess/visualization.cc

```diff
@@ -169,10 +169,10 @@ namespace aspect
 
     template <int dim>
     void
-    Visualization<dim>::write_master_files (const std::string &solution_file_prefix,
+    Visualization<dim>::write_master_files (DataOut<dim> &data_out,
+                                            const std::string &solution_file_prefix,
```

Comment:
I think you should be able to make the first argument `const` to make it clear that you're not modifying it.